docs/src/style.md

```diff
@@ -144,8 +144,148 @@ function foo(x)
 end
 ```
 
+#### Line length
 
-#### TODO: Line breaks
+Line lengths are a contentious issue. Our foremost goal is to maximize code
+readability. Very long line lengths can be hard to easily comprehend. However,
+arbitrarily enforcing a maximum line length (like 80 characters) inevitably
+leads to cases in which slightly longer lines (e.g. 81 characters) might be more
+readable.
+
+Therefore, aim to keep line lengths under 80 characters by breaking lines
+for maximum readability (examples are given in the [Line breaks](@ref) section),
+but don't treat this as a hard rule.
+
+We make exceptions for
+
+ - URLs
+ - path names
+ - long string constants not containing whitespace
+ - lines for which breaking over multiple lines worsens readability
```

Comment:
The automatic formatters don't make exceptions like this. I'm ok with this for now but when we have an autoformatter this exception will probably go away.